pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -2,22 +2,22 @@ schema = 2
 
 [operators.gates.native]
 
-CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
-ControlledPhaseShift   = { properties = [ "invertible",                 "differentiable" ] }
-ControlledQubitUnitary = { properties = [ "invertible",                                  ] }
-CRot                   = { properties = [ "invertible"                                   ] }
-CRX                    = { properties = [ "invertible",                 "differentiable" ] }
-CRY                    = { properties = [ "invertible",                 "differentiable" ] }
-CRZ                    = { properties = [ "invertible",                 "differentiable" ] }
-CSWAP                  = { properties = [ "invertible",                 "differentiable" ] }
-CY                     = { properties = [ "invertible",                 "differentiable" ] }
-CZ                     = { properties = [ "invertible",                 "differentiable" ] }
+CNOT                   = { properties = [ "invertible", "controllable", "differentiable" ] }
```

Comment:
Just a matter of definition; there is no kernel for C(CNOT) but we can apply it as C(PauliX) moving a target wire into the control wires. Is it what we mean?

Reply:
haha.. just testing the patch discussing this morning... it's now in a separate branch. Removed them all!